test/var_compatibility_test/effr_calc.meta

```diff
@@ -34,6 +34,25 @@
   type = real
   kind = kind_phys
   intent = in
+  optional = .true.
```

Comment:
should these optional attributes be "True" instead of ".true."?

Reply:
Yes. Will fix.

Reply:
Yup. Fixed